### ISSUE TITLE
Release version on unix and mac does not define NO_CONSOLE

### ISSUE THREAD

**Issue by p-himik** [goldendict/goldendict#1045]
Because of this, autostarted GoldenDict spams messages like
```
goldendict.desktop[7525]: erasing..
goldendict.desktop[7525]: erase done..
goldendict.desktop[7525]: one finished.
goldendict.desktop[7525]: erasing..
goldendict.desktop[7525]: erase done..
```
in the syslog.

**Comment by vedgy** [comment]
#530 eliminates most of the annoying warnings in log.

**Comment by vedgy** [comment]
Please close this issue as fixed in 1d6b5efeac5c1f91ff0db57014ad33dc17d36282.
